views/orders.py

```diff
@@ -95,19 +89,24 @@ def get(self, order_id):
             .where(Order.order_id == str(order_id))
         )
 
+        if not res:
+            return None, NOT_FOUND
+
+        orderitems = []
```

Comment:
intendo il fatto che hai salvato la lista in una variabile chiamata `orderitems`